lib/lte_link_control/lte_lc.c

```diff
@@ -108,10 +108,10 @@ static const char unlock_plmn[] = "AT+COPS=0";
 #endif
 /* Request eDRX to be disabled */
 static const char edrx_disable[] = "AT+CEDRXS=3";
+/* Request modem to enable eDRX */
+static char edrx_value[25] = {0};
```

Comment:
This string can't by the LTE spec be longer than 4 characters + null-termination, so we should not allocate more